source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -508,6 +508,17 @@ class LineSerializer(PbGenericSerializer):
     line_groups = LineGroupSerializer(many=True, display_none=False)
 
 
+class StopAreaEquipmentsSerializer(PbGenericSerializer):
+    stop_area = jsonschema.MethodField(schema_type=lambda: StopAreaSerializer(), display_none=False)
```

Comment:
I don't think that you need a `MethodField` here as `StopArea` is defined before.
Also you could put those in another file.

Reply:
okay @kinnou02, a simple `stop_area = StopAreaSerializer(display_none=False) ` work well
On the other hand, I don't know where you want to move it. Because `EquipmentDetailsSerializer` is already here, since a last PR.

Reply:
stop_area = jsonschema.MethodField(schema_type=lambda: StopAreaSerializer(), display_none=False)
can be replaced by
stop_area = StopAreaSerializer(display_none=False) as StopAreaSerializer is declared above